packages/coordinator/src/components/CsvDownloaderButton/CsvDownloaderButton.tsx

```diff
@@ -0,0 +1,41 @@
+import CsvDownloader from "react-csv-downloader";
+import { IColumn } from "react-csv-downloader/dist/esm/lib/csv";
+import Button from "../Button";
+
+export interface ZMCsvDownloaderButtonProps {
+  columns?: IColumn[];
+  data: { [key: string]: any }[];
+  fileName: string;
+  separator?: string;
+  title: string;
+  onClick?: () => void;
```

Comment:
With respect to the previous comment, I believe when a new component like this is added it should not have optional values (especially when there's currently only 1 caller).

Either a prop is required, or it should not exist for now

Reply:
onClick is required for the button component.
So why not expose it externally as well, rather than a stump*?
*onClick={()=>{}}